<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor ACOES: Anadir Transacciones</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCalendarWidget" name="calFechaEmision">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lFechaEmision">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de emision *</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cMoneda">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lMoneda">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Moneda *</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eCuantia">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lCuantia">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cuantia *</string>
    </property>
   </widget>
   <widget class="QLabel" name="lFormaPago">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Forma de pago</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cTipo">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>20</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lTipo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo *</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eFormaPago">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lDestino">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Destino</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eMotivo">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eDestino">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lMotivo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Motivo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lProyecto">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Proyecto</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eBeneficiario">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lBeneficiario">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Beneficiario</string>
    </property>
   </widget>
   <widget class="QLabel" name="lApadrinamiento">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Apadrinamiento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bAtras">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt; Atras</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bConfirmar">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>430</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirmar transaccion</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cProyecto">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cApadrinamiento">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
